SLHCUpgradeSimulations/Configuration/python/phase1TkCustoms.py

```diff
@@ -142,9 +142,6 @@ def customise_DQM(process,pileup):
     process.DQMOfflinePrePOG.remove(process.TrackingDQMSourceTier0)
     process.DQMOfflineTracking.remove(process.TrackingDQMSourceTier0Common)
 
-    # Pixel DQM needs to be updated for phase1
-    process.DQMOfflinePreDPG.remove(process.siPixelOfflineDQM_source)
-

```

Comment:
To note the obvious, this conflicts with #13987 (which I hope would get merged before this one).